client/src/containers/TodoCardList/Item/requests.js

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios'
+
+export default {
+  card: {
+    completion: {
+      create: async (card, responseHandler, errorHandler) => {
```

Comment:
@datarockets/cuties why do we need 2 endpoints for updating 1 resource `Card`?

Reply:
@DDKatch we have a pattern for manipulating of record completion\archivation

Here is a [ror example](https://github.com/datarockets/bevital/blob/master/config/routes.rb#L46)

The idea of routes splitting - system flexibility and single responsibility

Reply:
@AleksSenkou I don't think my proposal brake single responsibility. Our current model `Card` is easy to change because of little amount of properties and dependent logic. I understand 2 queries if we'll go with 2 resources aka `CompletedCard` and 'IncompletedCard'. But I think it's overengineering now, so we should use just UPDATE or go with 2 models.

Reply:
@DDKatch I see what you mean, can you provide an example of update endpoint in this case?

Reply:
@AleksSenkou we can send `{completed: true/false}`  from client as usual and check on server side (pseudo code): 
```
if completed
 card.update({completedAt: Time.now})
else
 card.update(completedAt: null)
```
More than that we don't need to carry about conversions to string on client side, just on backend, because client don't get information about that `changedAt` through checkbox properly. So keep it on back side looks more logical for me.

Reply:
@DDKatch pseudo code looks good, imagine (real situation I had previously) for /update: 
1) we have 3 columns in Card that have to be updated
2) + we want to create some dependencies for Card 
2) + we want to send an event that user updated card
3) + we want to update the completedAt field with special rules

Don't you think it will be complicated for one request?